testsuite/features/finishing/srv_debug.feature

```diff
@@ -14,3 +14,6 @@ Feature: Debug the server after the testsuite has run
 
   Scenario: Check the taskomatic logs on server
     Then the taskomatic logs should not contain errors
+
+  Scenario: Check for out of memory errors
+   Then the messages log should not contain out of memory errors
```

Comment:
the "log messages" sounds better for me 🤔

Reply:
It does, referring to the `messages` file is not particularly simple
I'll change to your suggestion